addons/hr_recruitment/wizard/applicant_refuse_reason.py

```diff
@@ -48,7 +54,26 @@ def action_refuse_reason_apply(self):
                 raise UserError(_("Email template must be selected to send a mail"))
             if not self.applicant_ids.filtered(lambda x: x.email_from or x.partner_id.email):
                 raise UserError(_("Email of the applicant is not set, email won't be sent."))
-        self.applicant_ids.write({'refuse_reason_id': self.refuse_reason_id.id, 'active': False})
+
+        refused_applications = self.applicant_ids
+        if self.duplicates_count and self.duplicates:
+            duplicate_domain = self.applicant_ids._get_similar_applicants_domain()
+            duplicate_domain = expression.AND([duplicate_domain, [('application_status', '!=', 'refused')]])
+            duplicates_ids = self.env['hr.applicant'].search(duplicate_domain)
+            refused_applications |= duplicates_ids
+            refuse_bodies = {}
+            for duplicate in duplicates_ids:
+                url = '/web#%s' % url_encode({
+                    'id': self.applicant_ids[0].id,
+                    'model': 'hr.applicant',
+                    'view_type': 'form',
+                })
+                message = _("Refused automatically because this application has been identified as a duplicate of %(link) .")
+                link_html = '<a href="{}">{}</a>'.format(url, self.applicant_ids[0].name)
+                refuse_bodies[duplicate.id] = Markup(message) % {'link': Markup(link_html)}
```

Comment:
Can you check this?
```suggestion
                message = _("Refused automatically because this application has been identified as a duplicate of %(link).",
		link= Markup("<a href=%s>%s</a>") % (url, self.applicant_ids[0].name))
```